src/c++/perf_analyzer/genai-perf/README.md

```diff
@@ -126,21 +126,22 @@ using the `--prompt-source` CLI option.
 
 When the dataset is synthetic you can specify the following options:
 * `--num-prompts`: The number of unique prompts to generate.
-* `--synthetic-tokens-mean`: The mean number of tokens of synthetic input data.
-* `--synthetic-tokens-stddev`: The standard deviation number of tokens of synthetic
+* `--synthetic-input-tokens-mean`: The mean number of tokens of synthetic input data.
+* `--synthetic-input-tokens-stddev`: The standard deviation of the number of tokens of synthetic
   input data.
-* `--synthetic-requested-output-tokens`: The number of output tokens to ask the model
-  to return in the response.
 * `--random-seed`: The seed used to generate random values.
 
 When the dataset is coming from HuggingFace you can specify the following
 options:
 * `--num-prompts`: The number of unique prompts to generate.
 * `--dataset`: HuggingFace dataset to use for benchmarking.
 
-You can optionally set additional model inputs with the following option:
+You can optionally set additional model inputs with the following options:
 * `--extra-inputs {input_name}:{value}`: An additional input for use with the model with a singular value,
 such as `stream:true` or `max_tokens:5`. This flag can be repeated to supply multiple extra inputs.
+* `--output-tokens-mean`: The mean number of tokens to request from the model.
+* `--output-tokens-stddev`: The standard deviation of the number of tokens to request from the model.
```

Comment:
Are these part of the --extra-inputs?
I think their proximity in the documentation makes it a bit unclear.

Reply:
Good catch. Updated the README to be more clear.